jbpm-flow/src/main/java/org/jbpm/process/core/datatype/impl/type/ObjectDataType.java

```diff
@@ -137,11 +137,12 @@ private Optional<Object> getObjectFromClass(final Object value) {
     }
 
     private static Collection<DateFormat> dateFormats = Arrays.asList(new SimpleDateFormat("yyyy-MM-ddHH:mm:ss"),
+            new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss"),
```

Comment:
What about also adding a blank for covering more scenarios?
`new SimpleDateFormat("yyyy-MM-dd HH:mm:ss"),`

Reply:
Thats is already covered, you can see by manually adding the test case